known.toml

```diff
@@ -169,13 +148,6 @@ relative_directory = "theme/"
 version = "0.0.1"
 leftwm_versions = "<0.3.0"
 
-[[theme]]
-name = "SpaceJelly"
-repository = "https://gitlab.com/leftwmthemes/space_jelly"
```

Comment:
Looks like this theme is back up, maybe can restore it or see why leftwm-theme rejects it: https://gitlab.com/leftwmthemes/space_jelly